app/controllers/bookings_controller.rb

```diff
@@ -24,8 +24,9 @@ def edit
   end
 
   def update
-    if @booking.save
-      redirect_to theme_path(@theme), notice: "Booking was successfully created."
+    @booking = Booking.find(params[:id])
+    if @booking.update(booking_params)
+      redirect_to theme_path(id: @booking.theme.id), notice: "Booking was successfully update."
     else
```

Comment:
Looks great for line 27 and 28. 
I think line 29 should be:
redirect_to theme_path(@booking.theme)
What do you think @vivien-z ?

Reply:
I think the code might work, because we should pass an id number as an argument for theme_path().
[note: we might not need the 'id: ' part in the bracket]
[@Rom1023 your suggestion is easier, just pass the instance as argument]
[I will finish the view, so we can test it, before merge the branch]

Reply:
Oh ok yeah I get it, my suggestion is kind of a shortcut.

Reply:
@vivien-z once you are done, please let me know if it is working so I can finish the pundit

Reply:
@Rom1023 just see your message. Yes, I tested. Both works (@booking.theme) and (@booking.theme.id)

Reply:
Oh ok thanks for letting me know